lib/publiccloud/provider.pm

```diff
@@ -496,12 +496,7 @@ sub terraform_apply {
     my $ret = script_run("TF_LOG=$tf_log terraform apply -no-color -input=false myplan", $terraform_timeout);
     $self->terraform_applied(1);    # Must happen here to prevent resource leakage
     unless (defined $ret) {
```

Comment:
Is this actually working?

I see that `lib/susedistribution` has `script_run_die_on_timeout` set to 1 (https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/susedistribution.pm#L49), so I don't think the code ever reaches this line.